tests/unit/amo/pages/ErrorPages/TestNotFoundPage.js

```diff
@@ -0,0 +1,13 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+
+import NotFound from 'amo/components/Errors/NotFound';
+import NotFoundPage from 'amo/pages/ErrorPages/NotFoundPage';
+
+describe(__filename, () => {
+  it('renders a NotFound component', () => {
+    const root = shallow(<NotFoundPage />);
+
+    expect(root.find(NotFound)).toHaveLength(1);
```

Comment:
This could use one other test to ensure that `errorCode` gets passed to `NotFound` when defined.

Reply:
Hmm, this is interesting. Although the original `NotFound` component does support an `errorCode` prop, it doesn't seem to do anything with that prop. Therefore, the new code that I based on the old code also doesn't do anything with that prop. I can add a test to make sure that `NotFound` does have a prop of `errorCode` with the value we passed, but why are we bothering to pass an `errorCode` when nothing is ever done with it? Is there some magic here, or is it just a pointless prop?

Reply:
It looks like the use of `errorCode` in `NotFound` was removed in https://github.com/mozilla/addons-frontend/commit/20e84191659cf1d65ba2a8b3b938cc859b2cc0b6#diff-1d56f4fe2d7a8815ee512837af91ff1e. I guess we should just remove the pro entirely now?

Reply:
I removed the prop in an extra commit, but I can revert that commit if you don't think we should have removed it.

Reply:
Ha. Good catch.